lib/rust/parser/engine/build.rs

```diff
@@ -0,0 +1,18 @@
+use std::fs::File;
+use std::io::prelude::*;
+use parser_definition::lexer::Lexer;
+use flexer::Flexer;
+
+
+/// Generates lexer engine and saves the result into a file `src/lexer-engine.rs`.
+/// The content of the generated file can be used trough the `include!` macro.
```

Comment:
```suggestion
/// Generates the lexer engine and saves the result into the file `src/lexer-engine.rs`.
/// 
/// The content of the generated file can be used through the `include!` macro.
```

Please use spell check! IntelliJ has it built in.